test/parallel/test-readline-keys.js

```diff
@@ -44,6 +44,39 @@ function addTest(sequences, expectedKeys) {
   assert.deepStrictEqual(keys, expectedKeys);
 }
 
+const addKeyIntervalTest = (sequences, expectedKeys, interval = 150,
+                            assertDelay = 150) =>
+  (next) => () => {
```

Comment:
Yes. Its there to run `runKeyIntervalTests` in sequence. Is there a better way to handle it?